### PR TITLE
 Clean up any empty season folders after deletion of associated files

### PR DESCRIPTION
**Pull request by fernandog**
For some old shows I had a lot of empty season folders: Season 1, Season 2, etc...

@ratoaq2 as I said the ideia

### CODE REVIEW

**Comment by fernandog** [comment]
@labrys so what do you think?

**Comment by labrys** [comment]
I'd like some clarification first.  How did you end up with the empty season folders?  If they are a result of a Medusa action, I would say cleaning up the folders should be done (but only post that action of course).  If this is just to clear empty folders as a library clean-up action post some external process then I would say that this is not necessary and should not be included.  Some users may have a use-case for leaving empty season folders.

**Comment by fernandog** [comment]
@labrys as in the PR name:
@p0psicles 
"Clean up any empty season folders **after deletion of associated files**"

when I watch the episode and delete it, medusa will remove any leftovers associated files. so "Season 1" folder (example) will be empty

**Comment by fernandog** [comment]
@p0psicles @labrys ?

File: sickbeard/tv.py [hunk 1]
```diff
@@ -1313,6 +1313,10 @@ def refresh_dir(self):
                                 logger.log(u"Could not delete associate file: {0}. Error: {1}".format(related_file, e),
                                            logger.WARNING)
 
+        # clean up any empty season folders after deletion of associated files
+        for sub_dir in ek(os.listdir, self.location):
+            helpers.delete_empty_folders(ek(os.path.join, self.location, sub_dir), self.location)
```
Comment (fernandog) [inline sickbeard/tv.py]: @medariox here its ok. using `ek`

**Pull request merged**